agent/engine/docker_task_engine.go

```diff
@@ -825,16 +825,16 @@ func (engine *DockerTaskEngine) createContainer(task *api.Task, container *api.C
 				task.Arn, container.Name, mderr)
 		}
 	}
-
+	createContainerBegin := time.Now()
 	metadata := client.CreateContainer(config, hostConfig, dockerContainerName, createContainerTimeout)
 	if metadata.DockerID != "" {
 		engine.state.AddContainer(&api.DockerContainer{DockerID: metadata.DockerID,
 			DockerName: dockerContainerName,
 			Container:  container}, task)
 	}
 	container.SetLabels(config.Labels)
-	seelog.Infof("Task engine [%s]: created docker container for task: %s -> %s",
-		task.Arn, container.Name, metadata.DockerID)
+	seelog.Infof("Task engine [%s]: created docker container for task: %s -> %s, took %s",
```

Comment:
imo, these logs can be `debug` level.

Reply:
We can leave this as `Info` level similar to pullImage time logging.

Reply:
ok, cool